examples/access-token-server/src/index.ts

```diff
@@ -0,0 +1,47 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2020 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import express from 'express'
+import { json } from 'body-parser'
+import { accessTokenRouter, statusRouter } from './routes'
+import { getSettings } from './shared/settings'
+
+/**
+ * Access token server
+ * Gets a Google access token for a service account. To access the access token
+ * the caller MUST provide a valid looker client_id and client_secret. The caller
+ * must also provide a scope for the access token.
+ */
+
+const settings = getSettings()
+const app = express()
+app.use(json())
+app.use(accessTokenRouter)
+app.use(statusRouter)
+
+app.listen(settings.port, () => {
+  console.log(`server listening on port ${settings.port}`)
```

Comment:
if anyone sees this it's a reminder to share this info for viewing logs on dev/prod from Vincent:

> Go here
1:05
https://account.activedirectory.windowsazure.com/r#/applications
1:06
Look for AWS Dev Read Only Console Access
1:06
then
1:07
Then go here
1:07
https://console.aws.amazon.com/cloudwatch/home?region=us-east-1#logStream:group=/ecs/hackathon-app
1:07
Click on the log stream with the most current timestamp
1:07
Or newest
1:07
Which looks like this one     
ecs/hackathon-app/b1d0c0fc-e3df-439d-b394-eafbe09ced17